--- urllib3-1.24.1/docs/conf.py.orig	2018-10-05 23:00:05.000000000 +0200
+++ urllib3-1.24.1/docs/conf.py	2018-12-02 09:35:52.593757249 +0100
@@ -27,7 +27,10 @@
 # Mock some expensive/platform-specific modules so build will work.
 # (https://read-the-docs.readthedocs.io/en/latest/faq.html#\
 #  i-get-import-errors-on-libraries-that-depend-on-c-modules)
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 
 class MockModule(mock.Mock):
--- urllib3-1.25.9/test/test_connection.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_connection.py	2020-06-07 22:20:50.841614447 +0200
@@ -1,5 +1,8 @@
 import datetime
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 import pytest
 
--- urllib3-1.25.9/test/test_queue_monkeypatch.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_queue_monkeypatch.py	2020-06-07 22:31:23.954851245 +0200
@@ -1,6 +1,9 @@
 from __future__ import absolute_import
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 import pytest
 
--- urllib3-1.25.9/test/test_response.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_response.py	2020-06-07 22:25:48.023337809 +0200
@@ -7,7 +7,10 @@
 from io import BytesIO, BufferedReader, TextIOWrapper
 
 import pytest
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import six
 
 from urllib3.response import HTTPResponse, brotli
--- urllib3-1.25.9/test/test_retry.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_retry.py	2020-06-07 22:21:45.327985936 +0200
@@ -1,5 +1,8 @@
 import datetime
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import time
 
--- urllib3-1.25.9/test/test_ssl.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_ssl.py	2020-06-07 22:22:15.301156891 +0200
@@ -1,4 +1,7 @@
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 from urllib3.util import ssl_
 from urllib3.exceptions import SNIMissingWarning
--- urllib3-1.25.9/test/test_util.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/test_util.py	2020-06-07 22:21:18.341465468 +0200
@@ -7,7 +7,10 @@
 import socket
 from itertools import chain
 
-from mock import patch, Mock
+try:
+    from mock import patch, Mock
+except ImportError:
+    from unittest.mock import patch, Mock
 import pytest
 
 from urllib3 import add_stderr_logger, disable_warnings
--- urllib3-1.25.9/test/contrib/test_pyopenssl.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/contrib/test_pyopenssl.py	2020-06-07 22:30:33.011793894 +0200
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
 import os
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 try:
--- urllib3-1.25.9/test/contrib/test_pyopenssl_dependencies.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/contrib/test_pyopenssl_dependencies.py	2020-06-07 22:31:07.291608185 +0200
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
 import pytest
 
-from mock import patch, Mock
+try:
+    from mock import patch, Mock
+except ImportError:
+    from unittest.mock import patch, Mock
 
 try:
     from urllib3.contrib.pyopenssl import inject_into_urllib3, extract_from_urllib3
--- urllib3-1.25.9/test/with_dummyserver/test_connectionpool.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/with_dummyserver/test_connectionpool.py	2020-06-07 22:20:19.205119170 +0200
@@ -6,7 +6,10 @@
 import warnings
 import pytest
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from .. import TARPIT_HOST, VALID_SOURCE_ADDRESSES, INVALID_SOURCE_ADDRESSES
 from ..port_helpers import find_unused_port
--- urllib3-1.25.9/test/with_dummyserver/test_https.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/with_dummyserver/test_https.py	2020-06-07 22:18:03.172522789 +0200
@@ -8,7 +8,10 @@
 import tempfile
 import warnings
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import trustme
 
--- urllib3-1.25.9/test/with_dummyserver/test_socketlevel.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/with_dummyserver/test_socketlevel.py	2020-06-07 22:18:29.705712380 +0200
@@ -44,7 +44,10 @@
 import shutil
 import ssl
 import tempfile
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 import pytest
 import trustme
