--- urllib3-1.24.1/docs/conf.py.orig	2018-10-05 23:00:05.000000000 +0200
+++ urllib3-1.24.1/docs/conf.py	2018-12-02 09:35:52.593757249 +0100
@@ -27,7 +27,10 @@
 # Mock some expensive/platform-specific modules so build will work.
 # (https://read-the-docs.readthedocs.io/en/latest/faq.html#\
 #  i-get-import-errors-on-libraries-that-depend-on-c-modules)
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 
 class MockModule(mock.Mock):
--- urllib3-1.26.2/test/test_connection.py.orig	2021-01-17 21:03:32.174782572 +0100
+++ urllib3-1.26.2/test/test_connection.py	2021-01-17 21:06:45.107070701 +0100
@@ -1,6 +1,9 @@
 import datetime
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 from urllib3.connection import RECENT_DATE, CertificateError, _match_hostname
--- urllib3-1.26.2/test/test_queue_monkeypatch.py.orig	2021-01-17 21:03:32.174782572 +0100
+++ urllib3-1.26.2/test/test_queue_monkeypatch.py	2021-01-17 21:07:04.446965928 +0100
@@ -1,6 +1,9 @@
 from __future__ import absolute_import
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 from urllib3 import HTTPConnectionPool
--- urllib3-1.26.2/test/test_response.py.orig	2021-01-17 21:03:32.174782572 +0100
+++ urllib3-1.26.2/test/test_response.py	2021-01-17 21:07:27.800172746 +0100
@@ -9,7 +9,10 @@
 from io import BufferedReader, BytesIO, TextIOWrapper
 from test import onlyBrotlipy
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import six
 
--- urllib3-1.26.2/test/test_retry.py.orig	2021-01-17 21:03:32.174782572 +0100
+++ urllib3-1.26.2/test/test_retry.py	2021-01-17 21:07:44.043418082 +0100
@@ -1,6 +1,9 @@
 import warnings
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 from urllib3.exceptions import (
--- urllib3-1.26.2/test/test_ssl.py.orig	2021-01-17 21:03:32.174782572 +0100
+++ urllib3-1.26.2/test/test_ssl.py	2021-01-17 21:07:59.646666886 +0100
@@ -1,6 +1,9 @@
 from test import notPyPy2
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 from urllib3.exceptions import SNIMissingWarning
--- urllib3-1.26.2/test/test_util.py.orig	2021-01-17 21:03:32.178115887 +0100
+++ urllib3-1.26.2/test/test_util.py	2021-01-17 21:08:29.376505825 +0100
@@ -9,7 +9,10 @@
 from test import notBrotlipy, onlyBrotlipy, onlyPy2, onlyPy3
 
 import pytest
-from mock import Mock, patch
+try:
+    from mock import Mock, patch
+except ImportError:
+    from unittest.mock import Mock, patch
 
 from urllib3 import add_stderr_logger, disable_warnings, util
 from urllib3.exceptions import (
--- urllib3-1.25.9/test/contrib/test_pyopenssl.py.orig	2020-04-16 14:42:30.000000000 +0200
+++ urllib3-1.25.9/test/contrib/test_pyopenssl.py	2020-06-07 22:30:33.011793894 +0200
@@ -1,7 +1,10 @@
 # -*- coding: utf-8 -*-
 import os
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 
 try:
--- urllib3-1.26.2/test/contrib/test_pyopenssl_dependencies.py.orig	2021-01-17 21:10:14.782601458 +0100
+++ urllib3-1.26.2/test/contrib/test_pyopenssl_dependencies.py	2021-01-17 21:12:17.745268645 +0100
@@ -1,6 +1,9 @@
 # -*- coding: utf-8 -*-
 import pytest
-from mock import Mock, patch
+try:
+    from mock import Mock, patch
+except ImportError:
+    from unittest.mock import Mock, patch
 
 try:
     from urllib3.contrib.pyopenssl import extract_from_urllib3, inject_into_urllib3
--- urllib3-1.26.2/test/with_dummyserver/test_connectionpool.py.orig	2021-01-17 21:10:14.782601458 +0100
+++ urllib3-1.26.2/test/with_dummyserver/test_connectionpool.py	2021-01-17 21:12:56.965056174 +0100
@@ -10,7 +10,10 @@
 from test import LONG_TIMEOUT, SHORT_TIMEOUT, onlyPy2
 from threading import Event
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import six
 
--- urllib3-1.26.8/test/with_dummyserver/test_https.py.orig	2022-03-10 06:29:38.588432315 +0100
+++ urllib3-1.26.8/test/with_dummyserver/test_https.py	2022-03-10 06:30:20.714870763 +0100
@@ -19,7 +19,10 @@ from test import (
     resolvesLocalhostFQDN,
 )
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import trustme
 
--- urllib3-1.26.2/test/with_dummyserver/test_socketlevel.py.orig	2021-01-17 21:10:14.782601458 +0100
+++ urllib3-1.26.2/test/with_dummyserver/test_socketlevel.py	2021-01-17 21:18:48.833149940 +0100
@@ -52,7 +52,10 @@
 )
 from threading import Event
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import pytest
 import trustme
 
