--- urllib3-1.26.2/test/appengine/test_urlfetch.py.orig	2021-01-17 21:19:07.753047443 +0100
+++ urllib3-1.26.2/test/appengine/test_urlfetch.py	2021-01-17 21:20:34.032580026 +0100
@@ -2,7 +2,10 @@
 App Engine sandbox enabled that urllib3 appropriately uses the App
 Engine-patched version of httplib to make requests."""
 
-import httplib
+try:
+    import httplib
+except ImportError:
+    import http.client as httplib
 import pytest
 import StringIO
 from mock import patch
